psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -247,9 +259,11 @@ def scf_iterate(self, e_conv=None, d_conv=None):
         self.set_energies("Total Energy", SCFE)
         Ediff = SCFE - SCFE_old
         SCFE_old = SCFE
-
         status = []
 
+        if smart_enabled:
```

Comment:
I would add a note that this will change the current SCF state so that the desired iterations are run. It is a bit transparent without this.